packages/api-client/src/api/serializers/product.ts

```diff
@@ -229,25 +228,3 @@ export const deserializeLimitedVariants = (apiProducts: IProducts): ProductVaria
     };
   });
 };
-
-const addHostToImage = (image, config: ApiConfig) => ({
```

Comment:
@fpdrozd I remember this had a specific purpose - in some cases the urls returned by the API were relative (without the host part). Is it safe to delete it now?

Reply:
This function is only used in getProduct and getProducts. In both of those cases the urls coming from the api include the host. If you aren't aware of any edge cases where the urls returned by those endpoints are relative then it's safe to remove that serializer completely.

Reply:
What I'm referring to is that it _used to_ return relative paths. I think that this behavior changed in the newer version of Rails, but we should support a scenario where the paths returned by the API are relative, otherwise we'll break it for e.g. Spree 4.4 which is still supported.

Reply:
Got it. Then we should keep that function and make it prepend the host only if it's not present. Makes sense?

Reply:
Yup, let's do that